src/array_api_stubs/_2022_12/fft.py

```diff
@@ -312,7 +312,7 @@ def irfft(
     Returns
     -------
     out: array
-        an array transformed along the axis (dimension) indicated by ``axis``. The returned array must have a real-valued floating-point data type determined by :ref:`type-promotion`. The length along the transformed axis is ``n`` (if given) or ``2*(m-1)`` (otherwise).
+        an array transformed along the axis (dimension) indicated by ``axis``. The returned array must have a real-valued floating-point data type whose precision matches the precision of ``x`` (e.g., if ``x`` is ``complex128``, then the returned array must have a ``float64`` data type). The length along the transformed axis is ``n``, if given or ``2*(m-1)`` otherwise.
```

Comment:
I think NumPy's `irfft` behavior is correct.
```suggestion
        an array transformed along the axis (dimension) indicated by ``axis``. The returned array must have a real-valued floating-point data type whose precision matches the precision of ``x`` (e.g., if ``x`` is ``complex128``, then the returned array must have a ``float64`` data type). The length along the transformed axis is ``n`` if given, or ``2*(m-1)`` otherwise. Therefore, to get an odd number of output points along ``axis``, ``n`` must be specified.
```

Reply:
Do you mean ``s`` must be specified?

Reply:
This is `irfft` not `irfftn` so I think it's `n` not `s`. 

<rant> we made a bad choice in the `s`/`n` argument names, NumPy was a bad model, is it too late to change? </rant>